packages/react/src/recipes/FlexTemplate/FlexSection/FlexSection.stories.tsx

```diff
@@ -0,0 +1,766 @@
+      if: {arg: 'showRivers', truthy: true},
+    },
+    riverImageTextRatio: {
+      control: {type: 'select'},
+      options: ['50:50', '60:40'],
+      description: 'Image to text ratio in river',
+      table: {category: 'River'},
+      if: {arg: 'showRivers', truthy: true},
+    },
+    riverHasLabel: {
+      control: 'boolean',
+      description: 'Show label in river',
+      table: {category: 'River'},
+      if: {arg: 'showRivers', truthy: true},
+    },
+    riverLabelColor: {
+      control: {type: 'select'},
+      options: LabelColors,
+      description: 'Color of the river label',
+      table: {category: 'River'},
+      if: {arg: 'riverHasLabel', truthy: true},
+    },
+    riverLabelSize: {
+      control: {type: 'select'},
+      options: ['small', 'medium', 'large'],
+      description: 'Size of the river label',
+      table: {category: 'River'},
+      if: {arg: 'riverHasLabel', truthy: true},
+    },
+    riverVisualType: {
+      control: {type: 'radio'},
+      options: ['image', 'video'],
+      description: 'Type of visual to display in river',
+      table: {category: 'River'},
+      if: {arg: 'showRivers', truthy: true},
+    },
+
+    enableRiverStoryScroll: {
+      control: 'boolean',
+      description: 'Enable river story scroll',
+      table: {category: 'River'},
+    },
+
+    // Visual settings controls
+    backgroundColor: {
+      control: {type: 'select'},
+      options: ['default', 'subtle'],
+      description: 'Section background color',
+      table: {category: 'Visual Settings'},
+    },
+    paddingBlockStart: {
+      control: {type: 'select'},
+      options: ['none', 'condensed', 'normal', 'spacious'],
+      description: 'Top padding of the section',
+      table: {category: 'Visual Settings'},
+    },
+    paddingBlockEnd: {
+      control: {type: 'select'},
+      options: ['none', 'condensed', 'normal', 'spacious'],
+      description: 'Bottom padding of the section',
+      table: {category: 'Visual Settings'},
+    },
+    colorMode: {
+      control: {type: 'select'},
+      options: ['inherit', 'light', 'dark'],
+      description: 'Color mode for the section',
+      table: {category: 'Visual Settings'},
+    },
+    verticalGap: {
+      control: {type: 'select'},
+      options: ['none', 'condensed', 'normal', 'spacious'],
+      description: 'Vertical gap between elements',
+      table: {category: 'Visual Settings'},
+    },
+    roundedCorners: {
+      control: 'boolean',
+      description: 'Enable rounded corners',
+      table: {category: 'Visual Settings'},
+    },
+    hasBorderBottom: {
+      control: 'boolean',
+      description: 'Show bottom border',
+      table: {category: 'Visual Settings'},
+    },
+  },
+}
+
+export default meta
+
+// @ts-expect-error As this is purely a demo component the types are not exact
+export const FlexSectionDefault: StoryFn<typeof FlexSection> = args => <FlexSection component={createMockData(args)} />
```

Comment:
Could we switch this to CSF3 please? We're currently trying to move over to CSF3 everywhere.